algorithm/classify.py

```diff
@@ -51,3 +59,45 @@ def get_redundant_data(self) -> dict[str:int]:
         Getter fn for redundant data
         """
         return self.redundant_data
+
+    def categorize_data(self):
+        """
+        Categorize the raw data into "name," "email," "birthday," and "cell_no" lists.
+        """
+
+        # Regex for cell numbers, birthdays, and emails
+        cell_no_regex = re.compile(r'^\+63-\d{10}$')
+        birthday_regex = re.compile(r'^\d{4}-\d{2}-\d{2}$')
+        email_regex = re.compile(r'^.+@.+\..{2,}$')
+
+        for data_set in self.raw_data:
+            for value in data_set:
+                value = value.strip()
+                
+                if cell_no_regex.match(value):
+                    self.categorized_data["cell_no"].add(value)
+                elif birthday_regex.match(value):
+                    self.categorized_data["birthday"].add(value)
+                elif email_regex.match(value):
+                    self.categorized_data["email"].add(value)
+                else:
+                    formatted_name = format_name(value)
+                    if formatted_name is not None:
```

Comment:
Maybe this can be removed since the output will always be `typeof string` naman.